### PR TITLE
Gidk 1b74u6 beta

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/webpack.partial.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
             'isCordova': JSON.stringify(false),
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
-            'AppUrl': JSON.stringify('https://beta.giddh.com/'),
```
Comment (ravinderwalkover) [inline apps/web-giddh/webpack.partial.js:12]: is it required to remove the trailing slash?

File: apps/web-giddh/webpack.partial.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = {
             'process.env.isElectron': JSON.stringify(false),
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
-            'process.env.AppUrl': JSON.stringify('https://beta.giddh.com/'),
```
Comment (ravinderwalkover) [inline apps/web-giddh/webpack.partial.js:24]: is it required to remove the trailing slash?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/webpack.partial.js (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/webpack.partial.js]: Updated

File: apps/web-giddh/webpack.partial.js (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/webpack.partial.js]: Updated

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**